### DISCUSSION TITLE
Missing "visibility" attribute from historical weather API

### DISCUSSION THREAD

**Discussion by sarthak** [open-meteo/open-meteo#805] [Q&A] [answered]
The "visibility" attribute seems missing from the [Historical Weather API](https://open-meteo.com/en/docs/historical-weather-api). This attribute is present in other endpoints, such as the [Forecast API](https://open-meteo.com/en/docs). I am wondering if this a fundamental limitation of historical models, or merely missing work?

**Comment by patrick-zippenfenig** [comment, marked as answer]
It is not available from the underlaying datasets IIRC. However, you can use the upcoming [historical forecast API](https://open-meteo.com/en/docs/historical-forecast-api) and select GFS global. The ICON domain will support visibility in the future.
